src/control-plane/backend/lambda/api/service/reporting.ts

```diff
@@ -1496,14 +1496,15 @@ export class ReportingService {
       const streamingStackIdPrefix =
       streamingStack?.StackId?.split('/')[2].split('-')[0];
       for (let appId of streamEnableAppIds) {
+        const streamName = `${SINK_STREAM_NAME_PREFIX}${pipeline.projectId}_${appId}_${streamingStackIdPrefix}`;
         enableConfigs.push({
           appId: appId,
-          streamArn: `arn:${awsPartition}:kinesis:${pipeline.region}:${awsAccountId}:stream/${SINK_STREAM_NAME_PREFIX}${pipeline.projectId}_${appId}_${streamingStackIdPrefix}`,
+          streamArn: `arn:${awsPartition}:kinesis:${pipeline.region}:${awsAccountId}:stream/${streamName.toLowerCase()}`,
```

Comment:
Kinesis allows `[A-Z]` as stream name. https://docs.aws.amazon.com/kinesis/latest/APIReference/API_CreateStream.html#Streams-CreateStream-request-StreamName

Is it our internal naming convention when creating the stream?

Reply:
yes. In streaming stack, the stream name is converted to lower case.